src/v2/spec.yaml

```diff
@@ -671,13 +671,17 @@ components:
           $ref: "#/components/schemas/Fid"
         hash:
           type: string
-    EmbedCastId:
+    EmbedCast:
```

Comment:
@Shreyaschorge will this result in a breaking change on the SDK?

Reply:
It is a breaking change, but chance of it causing a real problem for any user is small since they would have to be referring to the changed types by name. Either way, just including this breaking change warning in the changelog should be sufficient:
- `EmbeddedCast` type renamed to `Embed`
- `EmbedCastId` type renamed to `EmbedCast`

Reply:
@manan19 Yes it will be a breaking change but @tybook is right, it won't cause a big issue because these schemas to the properties that are a part of the Cast schema, and idt many devs will use these schemas